src/status.c

```diff
@@ -280,14 +280,19 @@ int git_status_list_new(
 	if ((error = git_repository__ensure_not_bare(repo, "status")) < 0 ||
 		(error = git_repository_index(&index, repo)) < 0)
 		return error;
-
-	/* if there is no HEAD, that's okay - we'll make an empty iterator */
-	if ((error = git_repository_head_tree(&head, repo)) < 0) {
-		if (error != GIT_ENOTFOUND && error != GIT_EUNBORNBRANCH)
-			goto done;
-		giterr_clear();
+	
+	if (opts != NULL && opts->head_tree != NULL) {
+		head = opts->head_tree;
 	}
-
+	else {
```

Comment:
We usually have closing braces and else branches on the same line.

Reply:
Oops, I missed this in my first round of fixes. Looking at other files, I don't know if "usually" is accurate :) but if it's preferred I can certainly make the change.

Reply:
I went ahead and did this one too.

Reply:
Well, sure. We're being inconsistent with many things, but at least I try to avoid introducing more inconsistencies ;) And in this case we're definitly being consistent most of the time with closing brace and else on the same line.

This still seems to not be fixed here -- did you forget to push your amends?